src/api/app/components/bs_request_description_component.rb

```diff
@@ -8,24 +8,24 @@ class BsRequestDescriptionComponent < ApplicationComponent
   delegate :requester_str, to: :helpers
   delegate :creator_intentions, to: :helpers
 
-  def initialize(bs_request:)
+  def initialize(bs_request:, links: false)
     super
     @bs_request = bs_request
+    @links = links
   end
 
   # rubocop:disable Metrics/CyclomaticComplexity
   # rubocop:disable Metrics/PerceivedComplexity
   # rubocop:disable Metrics/BlockLength
-  # rubocop:disable Style/FormatString
   def call
```

Comment:
To me, this function is too noisy, and rubocop agrees with me above this line. I would split it.